### ISSUE TITLE
Not enough strength to remove car part, while having a higher strength

### ISSUE THREAD

**Issue by Rayeneth** [CleverRaven/Cataclysm-DDA#54383]
### Describe the bug

I had 14 strength on my @ and attempted to remove an engine from a vehicle that required 13. It shown in red and I had no debuffs or strength related traits. 

### Steps to reproduce

I'm not sure if it can be reproduced as it occurred randomly, but here're the steps I experienced.
1) have 14 strength
2) try and remove an inline engine/reinforced glass (13 strength requirement)
3) be unable.

### Expected behavior

I expected to be able to unisntall an engine

### Screenshots

https://cdn.discordapp.com/attachments/642349261320880128/931342282077134858/unknown.png
https://media.discordapp.net/attachments/642349261320880128/931342326025027584/unknown.png?width=605&height=676

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 5662136 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Blaze Industries [blazeindustries],
    Generic Guns [generic_guns]
]

### Additional context

As far as I can tell everything else was normal. The problem persisted for days before I decided to use a forklift arm instead.

**Comment by LeahLuong** [comment]
It may be there's a tool req your failing to see. This happens to me all the time on Android because the button that's supposed to scroll the reqs menu doesn't work. Heavy vehicle parts often need a Lifting quality of some lvl. Hence, why it prob worked when you involved the forklift.

**Comment by coyo7e** [comment]
Yeah from the screenshot it appears that they ought to be able to do it with a good wrench and bare hands. Not sure which engine that is, though - I can't find it on the hitch-hiker's guide site and I know squat about engines - can you (e) examine it to see its weight and stuff?

**Comment by Rayeneth** [comment]
> Yeah from the screenshot it appears that they ought to be able to do it with a good wrench and bare hands. Not sure which engine that is, though - I can't find it on the hitch-hiker's guide site and I know squat about engines - can you (e) examine it to see its weight and stuff?
> 
> The other thing you that be stopping it is removing the alternator and/or other attached parts, first.. When you hit shift-P to check the log, what's the message it complains about? It's a long shot since you're in windows version so there shouldn't be anything cut off by the screen.
> 
> It is fairly easy to build a steel frame and some forklift forks, which iirc is about all you need for a lifting score.

It was the last thing left on the frame and the forklift let me remove it without additional removals. I don't have the save anymore (died) but it came off a "sports car"

**Comment by LeahLuong** [comment]
Maybe it's a Goldilocks situation where the coder left out a "<" somewhere making it so you need exactly 13 Strength.

**Comment by Rayeneth** [comment]
> Maybe it's a Goldilocks situation where the coder left out a "<" somewhere making it so you need exactly 13 Strength.

This is an interesting possibility, if I knew how to find the process I'd check. It also occurred later when I was trying to dismantle reinforced windshields.

**Comment by coyo7e** [comment]
I looked up a sports car in-game and raised my STR to 13 and got the same problem in current experimental. No feedback message - the game simply won't allow you to hit Enter on the item to attempt to remove, and I did remove the alternator first.

Raising STR to 14 does allow it to be removed normally without a lifting tool, so the < vs <= thing is likely what's up with it.

**Comment by lisareichelson** [comment]
Hi! I'm new to open source, and I have to work on an open source project for my SWE class. Can I work on this?

**Comment by LeahLuong** [comment]
Hey, if teach is cool w/ you playing a video game as a project then you have a cool teach.

**Comment by leemuar** [comment]
> Hi! I'm new to open source, and I have to work on an open source project for my SWE class. Can I work on this?

You don't have to ask. Just do it

**Comment by haveric** [comment]
From some recent testing, the strength check appears to be using the base strength and not the current/modified strength, such as from mutations. I need to dig into the code some more, but I should be able to fix this.
